integration-tests/create-react-app/src/App.percy.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
```

Comment:
The main annoying thing is that for non-ejected `create-react-app` projects, you cannot change any ESLint settings, and their Webpack config blows up on ESLint validation errors. In this file it complains that `percySnapshot` is not defined. Though we have an ESLint plugin package explicitly for whitelisting percy variables, there's no way to add it to `create-react-app`. So I've just added `eslint-disable` to this file.